website/content/partials/known-issues/1_16_azure-secrets-engine-client-id.mdx

```diff
@@ -0,0 +1,13 @@
+### Azure secrets engine role creation failing 
+
+#### Affected versions
+
+- 1.16.0, 1.16.1, 1.16.2
+
+#### Issue
+
+Creating Azure Secrets engine roles by specifying the Azure App Registration 
+Object ID as the application_object_id causes Vault to error out stating error 
+loading Application: no application found. This was introduced by 
+`vault-plugin-secrets-azure` v0.17.0, which incorrectly queries for an application
+by client_id and not application_object_id. 
```

Comment:
Is there a workaround? We should state either way about a workaround

Reply:
Yes there is. They could use the `client_id` instead of the `application_object_id`. However, they would need to remember to switch it back to the `application_object_id` once they upgrade to anything after 1.16.2. I'm going to add it because I think more info is better even if it is a little bit of a foot gun